home/tests/test_templates.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestHome:
+    def test_append_google_analytics_key(self, settings, client):
+        settings.GOOGLE_ANALYTICS_KEY = "UA-000000000-1"
+        response = client.get("/")
+        assert "UA-000000000-1" in str(response.content)
+
+
+@pytest.mark.django_db
+class TestAdmin:
+    def test_append_google_analytics_key(self, settings, admin_client):
+        settings.GOOGLE_ANALYTICS_KEY = "UA-000000000-1"
+        response = admin_client.get("/admin/")
+        assert "UA-000000000-1" in str(response.content)
+
+
+@pytest.mark.django_db
+class TestPanel:
+    def test_append_google_analytics_key(self, settings, admin_client):
+        settings.GOOGLE_ANALYTICS_KEY = "UA-000000000-1"
+        response = admin_client.get("/painel/")
```

Comment:
Apesar de estar na página `admin.py`, para acessar o painel nenhuma permissão deve ser requerida. Por isso aqui tu pode usar o `client` normal mesmo. Inclusive, talvez devamos renomear esse módulo pra deixá-lo mais claro para as pessoas...

```suggestion
        response = client.get("/painel/")
```

Reply:
Sim, vou alterar aqui. Neste caso não sei até se ficaria melhor criar outro arquivo de teste separado com a classe TestPanel no app de database...

Reply:
> Inclusive, talvez devamos renomear esse módulo pra deixá-lo mais claro para as pessoas...

Você diz renomear apenas o admin_client mesmo ou a classe também não está muito clara?

Reply:
Nah, só esse arquivo de testes mesmo serve. Eu estava falando sobre talvez renomear o `admin.py` (já que não temos bem um módulo de administrador mas sim um painel) mas isso é pra outro momento.